CHANGELOG.md

```
@@ -4,15 +4,2813 @@
+
+### Changed
+
+- Allow thumbnav items to shrink if `flex-wrap: nowrap` is set
+
+### Fixed
+
+- Fix Filter component height during transition
+- Fix Filter/Sortable component in IE11
+- Fix Overflow Auto component triggering `resize` event indefinitely
+- Fix Slider/Slideshow capture slide during transition in Safari 11
+- Fix Slider/Slideshow drag behavior in iOS 11.3+ (https://bugs.webkit.org/show_bug.cgi?id=184250)
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)